odoo/addons/test_read_group/tests/test_date_range.py

```diff
@@ -126,3 +126,114 @@ def test_duplicate_month(self):
         groups = self.Model.read_group(
             [('id', 'in', records.ids)], fields=['date', 'value'], groupby=['date:month', 'date:month'], lazy=False)
         self.assertEqual(groups, expected)
+
+
+class TestRelativeDateGranularity(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.Model = cls.env['test_read_group.fill_temporal']
+        cls.Model.create(({"value": "1", "date": "2021-02-09"},
+                          {"value": "20", "date": "2021-06-01"},
+                          {"value": "300", "date": "2022-02-01"}))
```

Comment:
In this context, using list brackets is less confusing than using tuple parentheses.
```suggestion
        cls.Model.create([{"value": "1", "date": "2021-02-09"},
                          {"value": "20", "date": "2021-06-01"},
                          {"value": "300", "date": "2022-02-01"}])
```